question.html

```diff
@@ -47,37 +47,15 @@
 		</div>
 		<div id="navbar" class="navbar-collapse collapse">
 			<ul class="nav navbar-nav">
-				<p class="navbar-text"><b>Ask, Answer, Learn.</b></p>
+				<p class="navbar-text">Question Room: <b>{{roomId}}</b></p>
 			</ul>
 			<ul class="nav navbar-nav navbar-right">
-				<li><a href="index.html" style="font-family: 'Bubblegum Sans', cursive">Home!</a></li>
-				<li>
-					<button type="button" class="btn btn-default navbar-btn" ng-hide="$authData" ng-click="FBLogin()">
-						Sign in with Facebook!
-					</button>
-				</li>
+				<li><a href="index.html">Change Room</a></li>
 			</ul>
 		</div><!--/.nav-collapse -->
 	</div>
 </nav>
 
-<div class="container">
```

Comment:
Yes, I am removing the jumbotron. 
It only contains two pieces of information which are relevant for the question page:
- The URL (which can be easily retrieved from the URL bar of every browser)
- The roomID (which is now displayed in the navigation bar)
The jumbotron blocks the view on the questions and is thus more of an obstacle than sth else.